Docs/Generali/glossario/01_content.tex

```diff
@@ -1,25 +1,42 @@
 % Insert content here
 \section{A}
+    \subsection{Action}
+    (s. c. f.)\\
+    Combinazione di script e configurazioni software che consentono di eseguire operazioni 
+    specifiche in risposta a determinati eventi, come modifiche al codice.
+    Vedi anche \nameref{GitHub Actions}.
     \subsection{Agile}
+    (agg.)
     Approccio alla gestione dei progetti che prevede la suddivisione in fasi e sottolinea 
     l'importanza della collaborazione e del miglioramento continuo tramite feedback. 
     Il team segue un ciclo di pianificazione, esecuzione e valutazione.
```

Comment:
Agile può essere anche un nome 

vedi (https://www.atlassian.com/it/agile)  Agile non è definito da una serie di.....